src/FileInput.js

```diff
@@ -1,14 +1,32 @@
 import React, { Component } from "react";
 import "./FileInput.css";
+const PapaParse = require('papaparse/papaparse.min.js');
 
 export default class FileInput extends Component {
+
+   handleChangeFile = e => {
+    for (let value of e.target.files) {
+        let reader  = new FileReader()
+        const filename = value.name;
+        reader.onload = event => {
+            const csvData = PapaParse.parse(
+              event.target.result,
+            );
+            this.props.onFileLoaded(csvData.data, filename);
+          };
+
+        reader.readAsText(value);
+    }
+  }
```

Comment:
it looks like papaparse can [parse the csv](https://www.papaparse.com/docs#local-files) without needing a separate FileReader instance, wondering if you gave any thought to this